devex/vscode-extension/src/common/utility.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import * as fse from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+import { AzureAccount, AzureSession } from "../typings/azure-account.api";
```

Comment:
Is this entire file Azure-specific?   Or does it combine Azure-specific and non-Azure-specific code?
If the former, then it would be better to rename the file, or put it under an azure subdirectory.
If the latter, then it would be better to split the file.  (Ok to do in a separate PR though.)

Reply:
Turns out the import is unused, so I've removed it. This file has a utility function to iterate over an array, used elsewhere for ACR support.